### PR TITLE
Specify profile fields for LinkedIn provider

### LINKED ISSUE #16559: LinkedIn Scope issue "InternalOAuthError: failed to fetch user profile"
We have been getting InternalOAuthErrors for linkedIn recently on PRODUCTION. 

Also reported by @IsaacAbrahamson in https://github.com/freeCodeCamp/freeCodeCamp/issues/14652#issuecomment-359528285

We need to update our config as per the suggested fix for the correct scope:
https://github.com/auth0/passport-linkedin-oauth2/issues/24#issuecomment-358279513

Just to note, this still links the profile, correctly, only thing is this exception needs to be handled, by correcting the scopes.

### PR DESCRIPTION
**Pull request by ojongerius**
Addresses #16559

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:
- [ ] Tested changes locally.
- [x] Addressed currently open issue .

Closes #16559

#### Description
See #16559 and https://github.com/auth0/passport-linkedin-oauth2/issues/24

### CODE REVIEW

File: config/secrets.js [hunk 1]
```diff
@@ -38,6 +38,14 @@ module.exports = {
     clientID: process.env.LINKEDIN_ID,
     clientSecret: process.env.LINKEDIN_SECRET,
     callbackURL: '/auth/linkedin/callback',
+    profileFields: [
```
Comment (ojongerius) [inline config/secrets.js]: These fields need a review.

Comment (raisedadead) [reply config/secrets.js]: I think, only the profile-url is needed for our purposes, and only adding that should be fine.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
Other than that we would need this change against our `backup/master` branch. Sorry I should have clarified that earlier.

**Comment by ojongerius** [comment]
@raisedadead  merging with `backup/master` was a little to painful, I'll close this and have raised https://github.com/freeCodeCamp/freeCodeCamp/pull/16715 which includes updated `profileFields`.

**Pull request closed without merging**